server/src/main/java/org/elasticsearch/monitor/fs/FsInfo.java

```diff
@@ -146,6 +201,19 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
                 builder.humanReadableField(Fields.AVAILABLE_IN_BYTES, Fields.AVAILABLE, getAvailable());
             }
 
+            if (effectiveLowStage != null) {
+                builder.field(Fields.EFFECTIVE_LOW_WATERMARK, getEffectiveLowWatermarkPercent());
+            }
+            if (effectiveHighStage != null) {
+                builder.field(Fields.EFFECTIVE_HIGH_WATERMARK, getEffectiveHighWatermarkPercent());
+            }
+            if (effectiveFloodStage != null) {
+                builder.field(Fields.EFFECTIVE_FLOOD_STAGE, getEffectiveFloodWatermarkPercent());
+            }
+            if (effectiveFrozenFloodStage != null) {
+                builder.field(Fields.EFFECTIVE_FROZEN_FLOOD_STAGE, getEffectiveFrozenFloodWatermarkPercent());
+            }
```

Comment:
I'd rather we emitted the bytes values instead of the percentages here (using `builder.humanReadableField`)

Reply:
... and the labels should be something like `low_watermark_free_space` (for the human-readable field) and `low_watermark_free_space_in_bytes` (for the raw number) etc.